selection_sort/java/selection_sort.java

```diff
@@ -0,0 +1,30 @@
+import java.util.Scanner;
+class Selection_sort
+{
+    static void selection_sort(int m[])
+    {   int a=m.length;
+        for(int i=0;i<a-1;i++)
+        {  int min=i;
+            for(int j=i+1;j<a;j++)
+            { if(m[j]<m[min])
+                min=j;
+            }
+            int aa=m[min];
+            m[min]=m[i];
+            m[i]=aa;
+        }
+    }
+    public static void main(String args[])
+    {
+        Scanner sc=new Scanner (System.in);
+        System.out.println("enter no. of elements in an array");
+        int a=sc.nextInt();
+        int arr[]=new int [a];
+        for (int i=0;i<a;i++)
+        arr[i]=sc.nextInt();
+        selection_sort(arr);
+        for(int mi:arr)
+         System.out.print(mi+" ");
+    }}
+    
+        
```

Comment:
Line contains following spacing inconsistencies:
- No newline at EOF.
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpyss_5621/selection_sort/java/selection_sort.java
+++ b/tmp/tmpyss_5621/selection_sort/java/selection_sort.java
@@ -27,4 +27,4 @@
          System.out.print(mi+" ");
     }}
     
-        +
```